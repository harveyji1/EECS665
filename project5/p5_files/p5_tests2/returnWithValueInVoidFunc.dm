fn : () void {
    return 5;
}

main : () void {
    fn();
}
