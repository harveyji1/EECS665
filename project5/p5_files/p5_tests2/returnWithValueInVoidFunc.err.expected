FATAL [2,12]-[2,13]: Return with a value in void function
Type Analysis Failed
